src/processing/vHexviewer/vHexviewer.cpp

```diff
@@ -23,15 +23,42 @@ class hexViewer : public RFModule, public Thread {
 
     virtual bool configure(yarp::os::ResourceFinder& rf)
     {
+
+        //display help
+        if(rf.check("help") || rf.check("h")) {
```

Comment:
Sorry, I commented days ago on that line of code and I thought you would receive a notification. I just saw that I could comment also here.

Reply:
thanks @marco-monforte. yeah I'm just using the resource finder style of command line arguments so it always has a `--` instead of a single `-`. For now it is fine, if we decide we need to make it more standard we can do so later.